draft-ietf-ppm-dap.md

```diff
@@ -804,19 +804,42 @@ the privacy guarantee is not affected by the sampling rate of the population,
 therefore a larger than expected batch size does not weaken the designed privacy
 guarantee.
 
-Implementation note: The goal for the Aggregators is to aggregate precisely
-`min_batch_size` reports per batch. Doing so, however, may be challenging for
-Leader deployments in which multiple, independent nodes running the aggregate
-interaction (see {{aggregate-flow}}) need to be coordinated. The maximum batch
-size is intended to allow room for error. Typically the difference between the
-minimum and maximum batch size will be a small fraction of the target batch size
-for each batch. If `max_batch_size` is not specified, the goal for Aggregators
-is to output the batch once it meets `min_batch_size`. How soon the batch should
-be output is deployment specific.
-
-[OPEN ISSUE: It may be feasible to require a fixed batch size, i.e.,
-`min_batch_size == max_batch_size`. We should know better once we've had some
-implementation/deployment experience.]
+Implementation note: The goal for the Aggregators is to aggregate the same
+number of reports in each batch. The target batch size is deployment-specific,
+and may be equal to or greater than the minimum batch size. Deciding how soon
+batches should be output is also deployment-specific. Exactly sizing batches may
+be challenging for Leader deployments in which multiple, independent nodes
+running the aggregate interaction (see {{aggregate-flow}}) need to be
+coordinated. The difference between the minimum batch size and maximum batch
+size is in part intended to allow room for error, and allow a range of target
```

Comment:
```suggestion
size is in part intended to allow room for rejecting some number of invalid reports, and allow a range of target
```

Reply:
Rejecting invalid reports isn't the only reason an aggregator could deviate from the target batch size, so I think David's more general text is better.

Reply:
Hmm, what else might happen?

Reply:
In this sentence, "room for error" was previously referring to variance in batch size that may occur depending on how the aggregation jobs are ran by a distributed system. (depending on implementation decisions, etc.) I added the "in part" caveat in this PR because we now also need to allow for rejecting invalid reports. I can rephrase "error" to be more clear here.

Reply:
Nah that's fine, thanks.